### PR TITLE
fix(nodejs): package fixes

### PR DESCRIPTION
**Pull request by sambacha**
- update github actions workflow 
- minor edits to npm package manifest file

Signed-off-by: sam bacha <sam@manifoldfinance.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,30 +19,41 @@
     "test-dev": "hardhat test --network dev",
     "ci": "hardhat compile && hardhat test"
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "keywords": [
+    "ethereum",
+    "account abstraction"
```
Comment (drortirosh) [inline package.json:24]: "erc 4337"

File: package.json [hunk 2]
```diff
@@ -2,6 +2,15 @@
   "name": "accountabstraction",
   "version": "1.0.0",
   "description": "",
+  "files": [
+    "/contracts/**/*.sol",
+    "/build/contracts/*.json",
```
Comment (drortirosh) [inline package.json:7]: we don't build with truffle, so we don't have a "build" folder at all (and neither contract/mocks)

File: package.json (hunk 1 above)
Comment (sambacha) [reply package.json:24]: added erc4337, a description would do well too

File: package.json (hunk 2 above)
Comment (sambacha) [reply package.json:7]: that and the `directories` field makes it redundant as well. Removed.

**Comment by sambacha** [comment]
FYI someone should take the package name on NPM, just in case of squatting / malicious user

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -1,21 +1,45 @@
...
+  FORCE_COLOR: 2
+
 jobs:
   test:
     runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ['14.x']
```
Comment (drortirosh) [inline .github/workflows/build.yml:21]: just wonder: is it any better to create a matrix-of-one-version ?
(this is not a node project. we only happen to write the solidity tests in javascript)

**Pull request closed without merging**